src/js/structs/NodesList.js

```diff
@@ -27,7 +25,7 @@ module.exports = class NodesList extends List {
 
       // Marathon API
       if (filters.ids) {
-        hosts = _.filter(hosts, function (host) {
+        hosts = hosts.filter(function (host) {
           return this.ids.indexOf(host.id) !== -1;
         }, {ids: filters.ids});
```

Comment:
Actually, you made the same change here: https://github.com/dcos/dcos-ui/pull/7/files/3fdaa6d8e495df77e379fc8a141c7e01b9880dd4#diff-b99d7a3b11dae3abc53eb56ab919aa0bR25
So would you mind doing it here too?